src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -26,33 +27,50 @@ object Scope {
     def getChildrenList(ast: Ast): Seq[Ast] = Option(ast.getAttribute("body")) collect {
       case list: AstList => list.asScala.toVector
     } getOrElse Seq.empty collect {
-      case ast: Ast => ast
+      case node: Ast
+        if node.getName == AstNodeName.Call ||
+          node.getName == AstNodeName.Scatter =>
+        node
+    }
```

Comment:
re the issue of generic-ness vs some knowledge, I see both sides on that one but we should only really be calling in the context of generating a workflow a single time an not re-calling it later down the road. Because of that I think if some specificity enhances other stuff it's a good idea